incubator/kafka/values.yaml

```diff
@@ -40,6 +40,11 @@ podManagementPolicy: OrderedReady
 rbac:
   enabled: false
 
+## Useful is using any custom authorizer
+secrets: {} # Pass in some secrets to mount and use (if required)
```

Comment:
It would be nice if this worked with both file-mounted secrets as well as secrets mounted by path in the env

Reply:
Need to update `README.md` with descriptions of new vars and defaults

Reply:
@benjigoldberg it is my understanding that this would work with my previous comment?  There might be a newer pattern that I may be less familiar with.